src/utils/constants.js

```diff
@@ -128,3 +128,15 @@ export const CLEAR_ALL = {
 	ALWAYS: 'always',
 	DEFAULT: 'default',
 };
+
+// search components modes
+export const SEARCH_COMPONENTS_MODES = {
+	SELECT: 'select',
+	TAG: 'tag',
+};
+
+export const searchComponentTypes = [
```

Comment:
@mohdashraf010897 There is already a util method present. search for `isSearchComponent`